Automattic/jetpack
# Stats Admin: Fix user capabilities

Related:  

- https://github.com/Automattic/jetpack/issues/31358
- https://github.com/Automattic/wp-calypso/pull/78210

## Proposed changes:

The PR proposes to add only the Jetpack required capabilities for Odyssey. Particularly `view_stats` is the one the PR want to add. It also adds the capabilities under site, to match Calypso state.

The capability list is duplicated from https://github.com/Automattic/jetpack/blob/1cd4309743d945d54948256370a5698cbafc51f0/projects/plugins/jetpack/sal/class.json-api-site-base.php#L888

### Other information:


## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:

* Open `/wp-admin/admin.php?page=stats`
* Open dev console
* Ensure `jetpackStatsOdysseyAppConfigData.intial_state.currentUser.capabilities` is an object with siteId as key and user capabilities as value
* Ensure `view_stats` is true when user is not admin but has role with access for Stats



<img width="493" alt="image" src="https://github.com/Automattic/jetpack/assets/1425433/ca6c5af5-dba3-4120-925a-fe9d01e1beff">
